tests/components/airzone/test_config_flow.py

```diff
@@ -53,13 +53,13 @@ async def test_form(hass):
 async def test_form_duplicated_id(hass):
     """Test setting up duplicated entry."""
 
+    entry = MockConfigEntry(domain=DOMAIN, data=CONFIG)
```

Comment:
We can do the same move in the init tests and the test util.

Reply:
Changed in 2060696e09bbf0f6374559024ccedd70cf500849

Reply:
Changed in 158c9b5c15592ab3b3df33a0eac7b8059179c680